__tests__/app.test.js

```diff
@@ -70,3 +70,57 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:articleid", () => {
+  test("responds with status code 200", () => {
+    return request(app).get("/api/articles/6").expect(200);
+  });
+  test("responds with an article object", () => {
+    return request(app)
+      .get("/api/articles/5")
+      .then(({ body }) => {
+        expect(typeof body).toBe("object");
+      });
+  });
+  test("responds with an article object with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .then(({ body }) => {
+        expect(body.article.hasOwnProperty("title")).toBe(true);
+        expect(body.article.hasOwnProperty("article_id")).toBe(true);
+        expect(body.article.hasOwnProperty("body")).toBe(true);
+        expect(body.article.hasOwnProperty("topic")).toBe(true);
+        expect(body.article.hasOwnProperty("created_at")).toBe(true);
+        expect(body.article.hasOwnProperty("votes")).toBe(true);
+        expect(body.article.hasOwnProperty("article_img_url")).toBe(true);
+      });
+  });
+  test("responds with the correct article object", () => {
+    const expected = {
+      article_id: 7,
+      article_img_url:
+        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      author: "icellusedkars",
+      body: "I was hungry.",
+      created_at: "2020-01-07T14:08:00.000Z",
+      title: "Z",
+      topic: "mitch",
+      votes: 0,
+    };
+    return request(app)
+      .get("/api/articles/7")
+      .then(({ body }) => {
+        expect(body.article).toEqual(expected);
+      });
+  });
```

Comment:
I'd probably make these specific assertions using the pattern you used on lines 89-95 but instead of hasOwnProperty you assert each property is exactly the value you want it to be. This is because you then leave the article object open to extension if the database changes and suddenly another property needs to go onto it

Reply:
I think I've changed this so it's as you described, not sure if I've understood this right though